submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+              type="text"
+              id="input_enter_name"
+              name=""
+              class="form-sign_pay_method_inputs input_enter_name"
+              placeholder="Enter your name"
+              minlength="5"
+              maxlength="20"
+              pattern="^[a-zA-Z]+$"
+              required
+            />
+            <span class="input_name title"></span>
+            <div class="wrapper_form-email_password">
+              <input
+                type="email"
+                placeholder="example@domaine.com"
+                name=""
+                class="form-sign_pay_method_inputs input_email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                required
+              />
+              <span class="input_name input_email_title"></span>
+              <input
+                type="password"
+                name=""
+                class="form-sign_pay_method_inputs input_password"
+                pattern="(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*"
+                title="Minimum 8 characters password,should have UPPERCASE, lowercase and Number"
+                required
+              />
+              <span class="input_name input_password_title"></span>
+            </div>
+            <div class="wrapper_form-choose_paysystem">
+              <input
+                type="radio"
+                name="payment"
+                id="visa_mastercard"
+                class="visually__hidden radio_input_payment"
+              />
+              <label for="visa_mastercard" class="form-choose_payment_radio">
+                <img
+                  src="images/mastercard.svg"
+                  alt="mastercards_icon"
+                  class="icon_paymethod"
+                />
+                <img
+                  src="images/visa.svg"
+                  alt="visa_icon"
+                  class="icon_paymethod"
+                />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="paypal"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="paypal"
+                class="form-choose_payment_radio paypal_label"
+              >
+                <img src="images/paypal.svg" alt="" class="icon_paypal" />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="amazon"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="amazon"
+                class="form-choose_payment_radio amazon_label"
+              >
+                <img
+                  src="images/amazon.svg"
+                  alt="amazon icon"
+                  class="icon_amazon"
+                />
+              </label>
+            </div>
+            <div class="wrapper_form-card_number">
```

Comment:
Check `cc-...` attributes for more comfortable using your form. [Read more](https://developer.mozilla.org/uk/docs/Web/HTML/Element/input#Attributes)

Reply:
Helpfull,thank's.Done.